data/settings.js

```diff
@@ -2300,52 +2300,58 @@ plugin {
 
 	/* notify-status plugin */
 
-	notify_status_dict: {
+	mailbox_notify_status: {
 		plugin: 'notify-status',
-		values: setting_types.STRING,
+		values: setting_types.BOOLEAN,
 		text: `
-The URI of the dictionary to use. This MUST be set for the plugin to be active.
+By default, all mailboxes are disabled.
 
-See [[link,dict]] for how to configure dictionaries.
+You can enable single mailboxes or mailbox wildcards by adding
```

Comment:
Also: "You can enable notifications for [...]"

Reply:
Non-grammatical suggestion: configuration descriptions should be identifying what the option does.

For example, for this setting, IMHO the description should be "Whether notifications for a mailbox are enabled."

This one should have a "default: no" entry.

With those two changes, I think the setting is adequately documented.  No need to worry about the language question (and it simplifies the description in general, which is good).

Keep the example, as it makes clear it can be set per-mailbox.